ptx/sec_antider.ptx

```diff
@@ -126,7 +126,7 @@
 
       <!-- Todo: Fix this weird image -->
 
-      <image width="100%">
+      <image xml:id="img_anti1" width="100%">
```

Comment:
I maybe forgot to alert you, but I'm working on chapter 5 now. So this may lead to a conflict. No big deal, but you could hold any more edits to the exercises in ch 5.

Reply:
There were about a dozen images that didn't have `xml:id`s. That's the change here.

Reply:
That also is the only change to Chapter 5. (didn't realize you were working on it)